### PR TITLE
chore(package.json): eslint autofixes minor issue

### PR DESCRIPTION
**Pull request by kwonoj**
- eslint for perf, spec will autofix minor issue
- bump up typescript to 1.8b

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "protractor": "2.2.0",
     "rx": "latest",
     "tslint": "2.5.0",
-    "typescript": "1.6.2",
+    "typescript": "1.8.0-dev.20151017",
```
Comment (kwonoj) [inline package.json]: relates to https://github.com/Microsoft/TypeScript/issues/5269#issuecomment-148947611 , bump up version including discussed changes.

File: package.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
     "build_test": "rm -rf dist/ && npm run lint && npm run build_es6 && npm run build_cjs && jasmine",
     "build_cover": "rm -rf dist/ && npm run lint && npm run build_es6 && npm run build_cjs && istanbul cover jasmine",
     "build_docs": "./docgen.sh",
-    "lint_perf": "eslint perf/",
-    "lint_spec": "eslint spec/**/*.js",
+    "lint_perf": "eslint perf/ --fix",
+    "lint_spec": "eslint spec/**/*.js --fix",
```
Comment (kwonoj) [inline package.json]: now let machine handle hard work, let people do easy part of writing code ;)

--fix option automatically corrects some of minor issues (spacing, etcs) but still some of lint error's won't be fixed. tslint does not have same options yet.

**Comment by staltz** [comment]
:+1:

**Comment by luisgabriel** [comment]
:+1:

**Pull request merged**